Source/Dafny/DafnyOptions.cs

```diff
@@ -538,6 +539,10 @@ public enum IncludesModes {
           ExtractCounterexample = true;
           return true;
 
+        case "printAllVerificationResults":
```

Comment:
I'm leaning toward making this a part of `/verificationLogger`, perhaps `/verificationLogger:text` or something like that.

Reply:
Yup, I'd suggest attaching the list of assertions for each batch to the corresponding `TestCase` using another custom property, just as we did for the `TestResult.ResourceCount`. If you squint you could imagine this concept applying to other kinds of test results too (i.e. the location and message of each test assertion such as `Assert.AreEqual` in an xUnit test method).

Then we can have the `csv` logger emit this data as a secondary file, or we can add a `/verificationLogger:console` option that will print out something like what you've got here. Or both!